appveyor.yml

```diff
@@ -43,14 +43,14 @@
               BOND_VS: "Visual Studio 12 2013"
               BOND_VS_NUM: 12
               BOND_ARCH: 32
-              BOND_BOOST: 58
+              BOND_BOOST: 61
```

Comment:
Appveyor does not seem to have 1.61 according to https://www.appveyor.com/docs/build-environment/#boost.